<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1312</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="binarysearchtreebutton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>271</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Binary Search Tree</string>
    </property>
   </widget>
   <widget class="QPushButton" name="binarysearchtreebutton_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>80</y>
      <width>271</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Red Black Tree</string>
    </property>
   </widget>
   <widget class="QPushButton" name="binarysearchtreebutton_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>271</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>AVL Tree</string>
    </property>
   </widget>
   <widget class="QPushButton" name="binarysearchtreebutton_4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>200</y>
      <width>271</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Phonebook</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1312</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
